translation/dest/variantTitle/en-US.xml

```diff
@@ -39,4 +39,6 @@
   <string name="go19x19">Surround the largest area(s) of the 19 by 19 board with your stones to win</string>
   <string name="backgammon">Get all your pieces into your home and then bear them off first to win</string>
   <string name="nackgammon">Get all your pieces into your home and then bear them off first to win</string>
+  <string name="breakthroughtroyka">The first player who makes a queen wins</string>
```

Comment:
Update this description to say something like "reach opponent rank to win"

Reply:
reach opponents home rank, or reach the final row